### PR TITLE
fix some typo of cinder.go and cinder_test.go

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
fix some typo of cinder go file
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglingxia** [comment]
/assign @FengyunPan 
thanks

**Review by FengyunPan — COMMENTED** [review]
Can you just update L165?

File: pkg/volume/cinder/cinder_test.go [hunk 1]
```diff
@@ -110,10 +110,7 @@ func (fake *fakePDManager) DetachDisk(c *cinderVolumeUnmounter) error {
 
 	// "Detach" the fake "device"
 	err = os.RemoveAll(fakeDeviceName)
-	if err != nil {
```
Comment (FengyunPan) [inline pkg/volume/cinder/cinder_test.go]: This is ok, no need to update.

**Pull request closed without merging**